resources/views/community/components/claims/finished-claim-table-row.blade.php

```diff
@@ -12,8 +12,16 @@
 ?>
 
 <tr>
-    <td class="pr-0">{!! gameAvatar($claim, label: false, tooltip: false) !!}</td>
-    <td class="w-full">{!! gameAvatar($claim, icon: false, tooltip: false) !!}</td>
+    <td class="py-1.5">
+        <x-game.multiline-avatar
+            :gameId="$claim['ID']"
```

Comment:
This needs to be the game ID, not the claim ID.

Master (castlevania=1462):
![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/0072a2bc-a154-449c-a9ce-a37720961775)

This branch:
![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/ca522df1-4210-433d-8af6-c9aa60de42bd)

Reply:
Thanks, fixed in https://github.com/RetroAchievements/RAWeb/pull/1681/commits/0e2f4c7f3c9d81138983bb1f36943dd5b13c62df.